polarCoordinateTraversal_dotprod.m

```diff
@@ -0,0 +1,198 @@
+function [radial_voxels, angular_voxels] = polarCoordinateTraversal(min_bound, max_bound, ray_origin, ray_direction, circle_center, ...
```

Comment:
re-name the function as well to ```polarCoordinateTraversal_dotprod.m```

Reply:
OK. Should I do this and then send a different request?

Reply:
Nope, you can do it in the same file and just resolve when its done, or if you're unsure if that's what was wanted (i.e. its nontrivial) then say "Done" and the reviewer will resolve. At least that's how I'm used to doing CRs.